index.html

```diff
@@ -0,0 +1,85 @@
+                    <option value="learning">Full Time Learner</option>
+                    <option value="notPrefered">Prefer not to say</option>
+                    <option value="other">Other</option>
+      
+                </select>
+            </div>
+            <div class="forminfo">
+                <p>Would you recommrn freeCodeCamp to a friend?</p>
+                <label><input name="user-recommend" value="definitely" type="radio" class="input-radio" checked/>Definitely</label>
+                <label><input name="user-recommend" value="Maybe" type="radio" class="input-radio" />Maybe</label>
+                <label><input name="user-recommend" value="not-sure" type="radio" class="input-radio" />Not sure</label>
+
+            </div>
+            <div class="forminfo">
+                <p>What is your favourite featu?re of freecCodeCamp</p>
+                <select id="favourite" name="favourite" class="forms" required>
+                    <option disabled selected value>Select an option</option>
+                    <option value="challenges">Challenges</option>
+                    <option value="projects">Projects</option>
+                    <option value="community">Community</option>
+                    <option value="openSource">Open Source</option>
+                  </select>
+            </div>
+            <div class="forminfo">
+                <p>What would you like to see improved? <span class="question">(Check all that apply)</span></p>
+                <label><input name="prefer" value="front-end-projects" type="checkbox" class="input-checkbox"/>Front-end Projects</label>
+                <label><input name="prefer" value="back-end-projects" type="checkbox" class="input-checkbox"/>Back-end Projects</label>
+                <label><input name="prefer" value="data-visualization" type="checkbox" class="input-checkbox"/>Data Visualization</label>
+                <label><input name="prefer" value="challenges" type="checkbox" class="input-checkbox"/>Challenges</label>
+                <label><input name="prefer" value="open-source-community" type="checkbox" class="input-checkbox"/>Open Source Community</label>
+                <label><input name="prefer" value="gitter-help-rooms" type="checkbox" class="input-checkbox"/>Gitter help rooms</label>
+                <label><input name="prefer" value="videos" type="checkbox" class="input-checkbox"/>Videos</label>
+                <label><input name="prefer" value="city-meetups" type="checkbox" class="input-checkbox"/>City Meetups</label>
+                <label><input name="prefer" value="wiki" type="checkbox" class="input-checkbox"/>Wiki</label>
+                <label><input name="prefer" value="forum" type="checkbox" class="input-checkbox"/>Forum</label>
+                <label><input name="prefer" value="additional-courses" type="checkbox" class="input-checkbox"/>Additional Courses</label>
+            </div>
```

Comment:
you can use a list for your labels, your div class info can be presented better if you use an unordered list, for example, that way you can take advantage of certain features lit flexbox